flask_accepts/decorators/decorators.py

```diff
@@ -367,7 +367,14 @@ def _model_from_parser(model_name: str, parser: reqparse.RequestParser) -> Model
 
 
 def merge(first: dict, second: dict) -> dict:
-    return {**first, **second}
+    for key, value in first.items():
```

Comment:
Perform a deep merge, which will also merge the response. This method is only called on line 370, where the given response data will be combined with any existing `__apidoc__` responses.